### PR TITLE
Minor rewrite of Symlink section

### PR DESCRIPTION
**Pull request by daniel-bogusz95**
Rewrite Symlink section to group information and avoid repetition,
and minor corrections to grammar (eliminated nominalization).

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [x] Documentation updated
- [ ] Unit tests added/updated

### CODE REVIEW

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :heavy_check_mark: |
| mint-pools.sh | :heavy_check_mark: |
Deleting image on docker hub
Deleting image locally

File: docs/gateway/nas.md [hunk 1]
```diff
@@ -96,17 +96,16 @@ NAS gateway implementation allows symlinks on regular files,
 
 ### Behavior
 
-- For reads symlink resolves to file symlink points to.
+- For reads symlinks resolve to the file the symlink points to.
```
Comment (harshavardhana) [inline docs/gateway/nas.md:99]:
```suggestion
- For reads symlinks resolve to the file the symlink points to. 
```

This wording is wrong - `symlink` is single here and `resolves` is correct.

**Review by harshavardhana — CHANGES_REQUESTED** [review]
rest of the words look good - please send these doc changes to grammar in a single PR - thanks @daniel-bogusz95

**Comment by harshavardhana** [comment]
Merged into single PR https://github.com/minio/minio/commit/ce6c23a360090fee242b6ce1fd490c5035ad78c0

**Pull request closed without merging**